bin/doctest

```diff
@@ -1,15 +1,22 @@
 #!/usr/bin/env node
 
-// Based on:
-// https://github.com/jashkenas/coffeescript/blob/1.9.1/src/command.coffee#L431-L441
-var args = process.argv.slice(1);
+var path = require('path');
+
+var args = process.argv.slice(2);
 var idx = args.indexOf('--nodejs');
-if (idx < 0 || idx === args.length - 1) {
-  require('../lib/command');
-} else {
-  require('child_process').spawn(
-    process.execPath,
-    args[idx + 1].split(/\s+/).concat(args.slice(0, idx), args.slice(idx + 2)),
-    {cwd: process.cwd(), env: process.env, stdio: [0, 1, 2]}
-  ).on('exit', process.exit);
-}
+var esm = Number(process.versions.node.split('.')[0]) >= 9;
+var flags = idx >= 0 && idx < args.length - 1;
+
+var nodeargs = []
+  .concat(flags ? args[idx + 1].split(/\s+/) : [])
```

Comment:
```suggestion
  .concat(flags ? [args[idx + 1].split(/\s+/)] : [])
```

Does GitHub's suggestion feature support changing multiple (consecutive) lines at once?

Reply:
Ignore this suggestion. I just realized that the expression will produce an array when evaluated. :)